src/Context/ThemeContext.tsx

```diff
@@ -14,7 +14,8 @@ export const ThemeContext = createContext<ITheme | null>(null);
 
 const ThemeContextProvider: React.FC<ThemeContextProviderProps>= (props)=>{
     const [isDarkMode, setIsDarkMode] = useState(() => {
```

Comment:
The bug here is that when we store the value in the localStorage , it will get converted to a string.
so we get "true" or "false" instead of true or false.
To solve this issue we can use JSON.parse as follows to get its boolean value:

```
  const [isDarkMode, setIsDarkMode] = useState(() =>localStorage.getItem("isDarkMode") );
```

Reply:
done